src/components/Schedule/SearchBar.tsx

```diff
@@ -1,26 +1,70 @@
 import React, { useState } from "react";
 import { CloseIcon, SearchIcon } from "@chakra-ui/icons";
 import {
+  Flex,
+  Box,
+  FormLabel,
   IconButton,
   Input,
   InputGroup,
   InputRightElement,
+  Radio,
+  RadioGroup,
+  Stack,
 } from "@chakra-ui/react";
 import { CourseSection } from "../../stores/schedule";
+import { Text } from "devextreme-react/linear-gauge";
+// import { RadioGroup } from "devextreme-react";
 
 interface SearchBarProps {
   getTermData: () => CourseSection[] | undefined;
   setScheduleData: (data: CourseSection[]) => void;
 }
-
+var courseFilter = "all";
+var termFilter = "all";
 export const SearchBar = (props: SearchBarProps) => {
   const { getTermData, setScheduleData } = props;
 
-  const [searchInput, setSearchInput] = useState("");
+  let [searchInput, setSearchInput] = useState("");
```

Comment:
Is there a reason this was changed to let instead of const? You should be able to use setSearchInput in cases where you want to change the value of searchInput (eg. on line 48).